src/event/handlers/LogMessageBulkDeleteHandler.ts

```diff
@@ -7,10 +7,10 @@ class LogMessageBulkDeleteHandler extends LogMessageDeleteHandler {
 	}
 
 	async handle(messages: Collection<Snowflake, Message>): Promise<void> {
-		for (const message of messages.array()) {
+		messages.forEach(async message => {
 			await super.sendLog(message);
-		}
+		});
```

Comment:
You could simplify this even more to: `await Promise.all(messages.map(super.sendLog))`.